Recrutify/Recrutify.Host/Controllers/OData/ProjectsController.cs

```diff
@@ -1,13 +1,12 @@
-﻿using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.OData.Query;
-using Microsoft.AspNetCore.OData.Routing.Controllers;
+﻿using System.Linq;
+using Microsoft.AspNet.OData;
+using Microsoft.AspNet.OData.Routing;
 using Recrutify.Services.DTOs;
 using Recrutify.Services.Services.Abstract;
 
 namespace Recrutify.Host.Controllers.OData
 {
+    [ODataRoutePrefix("Projects")]
     public class ProjectsController : ODataController
```

Comment:
try to rename our usual controller to Projects Controller - then in swagger both controllers should appear as one

Reply:
Renamed our regular controller in Projects Controller - doesn't work
Projects Controller.
Hides OData Projects Controller